### ISSUE TITLE
 WARN  node_modules/polished/lib/color/rgb.js (6:59) A comment                                                                                           15:14:25  "/*#__PURE__*/"  in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.

### ISSUE THREAD

**Issue by FezVrasta** [styled-components/polished#642]
- `polished` version: 4.2.2
- `JSS-in_CSS` library and version: irrelevant
- Any relevant JS library and version: Vite 5.0.0

## Mixin/Helper/Shorthand Usage

N/A

## What You Are Seeing

During builds, I get these warnings:

```

 WARN  node_modules/polished/lib/color/rgb.js (6:59) A comment                                                                                           15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/rgb.js (8:56) A comment                                                                                           15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/rgb.js (10:51) A comment                                                                                          15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/parseToRgb.js (6:53) A comment                                                                                    15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/parseToRgb.js (8:54) A comment                                                                                    15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/parseToRgb.js (10:51) A comment                                                                                   15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/rgba.js (6:55) A comment                                                                                          15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/rgba.js (8:48) A comment                                                                                          15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/rgba.js (10:51) A comment                                                                                         15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/transparentize.js (6:50) A comment                                                                                15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/transparentize.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/transparentize.js (8:50) A comment                                                                                15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/transparentize.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/transparentize.js (10:49) A comment                                                                               15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/transparentize.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.


 WARN  node_modules/polished/lib/color/transparentize.js (12:55) A comment                                                                               15:14:25

"/*#__PURE__*/"

in "node_modules/polished/lib/color/transparentize.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.

```

## What You Expected To See

No warnings.

**Comment by cherryeclipseworks** [comment]
It's happening on another parts of the polished package also:

```
node_modules/polished/lib/internalHelpers/_hslToHex.js (6:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/internalHelpers/_hslToHex.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/internalHelpers/_hslToHex.js (8:59) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/internalHelpers/_hslToHex.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/internalHelpers/_hslToHex.js (10:56) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/internalHelpers/_hslToHex.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/hsl.js (6:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/hsl.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/hsl.js (8:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/hsl.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/hsla.js (6:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/hsla.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/hsla.js (8:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/hsla.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/hsla.js (10:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/hsla.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgb.js (6:59) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgb.js (8:56) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgb.js (10:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/parseToRgb.js (6:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/parseToRgb.js (8:54) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/parseToRgb.js (10:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToRgb.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/parseToHsl.js (6:55) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToHsl.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/parseToHsl.js (8:53) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/parseToHsl.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgba.js (6:55) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgba.js (8:48) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/rgba.js (10:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/rgba.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/toColorString.js (6:48) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/toColorString.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/toColorString.js (8:49) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/toColorString.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/toColorString.js (10:48) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/toColorString.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/toColorString.js (12:49) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/toColorString.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/toColorString.js (14:51) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/toColorString.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/lighten.js (6:50) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/lighten.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/lighten.js (8:50) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/lighten.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/lighten.js (10:55) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/lighten.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
node_modules/polished/lib/color/lighten.js (12:58) A comment

"/*#__PURE__*/"

in "node_modules/polished/lib/color/lighten.js" contains an annotation that Rollup cannot interpret due to the position of the comment. The comment will be removed to avoid issues.
```

**Comment by FezVrasta** [comment]
I suppose this library is dead?

**Comment by bhough** [comment]
Should be addressed by #643.

**Comment by bhough** [comment]
This should now be addressed (in a non-breaking manner) with #647.
